spacewalk/admin/uyuni-update-config

```diff
@@ -4,32 +4,48 @@
 import sys
 import os.path
 import uuid
+import subprocess
 from spacewalk.common.rhnConfig import initCFG, CFG
+from spacewalk.satellite_tools.syncLib import log
 
 initCFG('server.susemanager')
-try:
-    if CFG.scc_backup_srv_usr:
-        # nothing to do
-        sys.exit(0)
-except AttributeError:
-    # key does not exist, we need to create it
-    pass
-
-scc_cred_file = "/etc/zypp/credentials.d/SCCcredentials"
-
-uuidNum = None
-if os.path.exists(scc_cred_file):
-    with open(scc_cred_file, "r") as f:
-        for line in f:
-            if line.startswith("username"):
-                _k, v = line.split("=", 2)
-                uuidNum = v.strip()
-                break
-if not uuidNum:
-    # scc expects either a SCC machine login (must exists in SCC)
-    # or a UUID4 following rfc4122 to identify a anonyme proxy
-    uuidNum = str(uuid.uuid4())
-with open("/etc/rhn/rhn.conf", "a") as r:
-    r.write("\n")
-    r.write("server.susemanager.scc_backup_srv_usr = {}\n".format(uuidNum))
+
+def initSccLogin():
+
+    try:
+        if CFG.scc_backup_srv_usr:
+            # nothing to do
+            return
+    except AttributeError:
+        # key does not exist, we need to create it
+        pass
+
+    scc_cred_file = "/etc/zypp/credentials.d/SCCcredentials"
+
+    uuidNum = None
+    if os.path.exists(scc_cred_file):
+        with open(scc_cred_file, "r") as f:
+            for line in f:
+                if line.startswith("username"):
+                    _k, v = line.split("=", 2)
+                    uuidNum = v.strip()
+                    break
+    if not uuidNum:
+        # scc expects either a SCC machine login (must exists in SCC)
+        # or a UUID4 following rfc4122 to identify a anonyme proxy
+        uuidNum = str(uuid.uuid4())
+    with open("/etc/rhn/rhn.conf", "a") as r:
+        r.write("\n")
+        r.write("server.susemanager.scc_backup_srv_usr = {}\n".format(uuidNum))
+
+def importSumaGPGKeyring():
+
+    result = subprocess.run(["import-suma-build-keys.sh"], capture_output=True, text=True)
```

Comment:
I'm probably missing something but does the script ends ".sh"? The file name doesn't have it and the spec file also doesn't. How is it computed?

Reply:
That happens when you do a last minute rename ... Thanks for finding.